### PR TITLE
Update integration test to check environment variables to enable alternative API URLs

### PR DESCRIPTION
**Pull request by smartyjohn**
This allows integration tests to run against other API sources aside from `*api.smartystreets.com` by specifying specific environment variables as listed in `CHANGELOG,md`.  This allows testing against standalone installations, or for running the SDK against qa/development servers.  The `docker-compose.yaml` is also modified so that integration tests can be run from a docker instance.

Note that this only affects the integration test specifically.

### CODE REVIEW

**Comment by smartyjohn** [comment]
I should add that this mirrors a similar change in the [dotnet SDK](https://github.com/smartystreets/smartystreets-dotnet-sdk/commit/2a2311f46f4e7979aba2409e9e5c9bad7aeaba87).

File: docker-compose.yml [hunk 1]
```diff
@@ -7,3 +7,8 @@ services:
     environment:
       - SMARTY_AUTH_ID
       - SMARTY_AUTH_TOKEN
+      - SMARTY_URL_INTERNATIONAL
```
Comment (mdwhatcott) [inline docker-compose.yml]: To future-proof this variable, I recommend we use `SMARTY_URL_INTERNATIONAL_STREET` here.

Comment (smartyjohn) [reply docker-compose.yml]: Updated.

**Review by MouaYing — APPROVED** [review]
Looks good to me.

**Comment by mdwhatcott** [comment]
LGTM

**Review by mdwhatcott — APPROVED** [review]

**Pull request merged**